pkg/api/addons_test.go

```diff
@@ -65,9 +65,12 @@ func TestPodSecurityPolicyConfigUpgrade(t *testing.T) {
 	isUpgrade := true
 	base64DataPSP := "cHNwQ3VzdG9tRGF0YQ=="
 	o.OrchestratorType = Kubernetes
-	o.KubernetesConfig.EnablePodSecurityPolicy = to.BoolPtr(true)
-	o.KubernetesConfig.PodSecurityPolicyConfig = map[string]string{
-		"data": base64DataPSP,
+	o.KubernetesConfig.Addons = []KubernetesAddon{
```

Comment:
Slightly academic, but keeping this UT around and converting it to make sense in the new pod-security-policy addons context